ParentWorkspace/SpringModule/src/main/java/com/miniproject/model/Item.java

```diff
@@ -0,0 +1,129 @@
+package com.miniproject.model;
+
+import java.math.BigDecimal;
+import java.sql.Blob;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToMany;
+import javax.persistence.Table;
+
+/**
+ * @author Mohaneswaran
+ *
+ *         maps with Category table with many to one cardinality.
+ */
+
+@Entity
+@Table(name = "ITEM")
+public class Item {
+	
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@Column(name = "ITEMID")
+	private Integer itemId;
+	
+	@ManyToMany
+	@JoinColumn(name = "CID", nullable = false)
+	private Category cId;
+	
+	@Column(name = "ITEM")
+	private String item;
+	
+	@Column(name = "BRANDNAME")
+	private String brandName;
+	
+	@Column(name = "COST")
+	private BigDecimal cost;
+	
+	@Column(name = "MEASURE")
+	private String measure;
+	
+	@Column(name = "OFFER")
+	private BigDecimal offer;
+	
+	public Item(Integer itemId, Category cId, String item, String brandName, BigDecimal cost, String measure,
+			BigDecimal offer, Blob image) {
```

Comment:
what is the use of Blob class/Data Type, and also u haven't declared image column to the item table.